### ISSUE TITLE
65A1 - Group Project

### ISSUE THREAD

**Issue by summerysaturn** [summerysaturn/y1-university#13]
Sup! I'm gonna be sharing this.

So at the moment we have an initial idea of style (BDG Unravelled), we have two potential subjects (Ubisoft and Activision/Blizzard). What we need at the moment is:

- [x] Decide on the company
- [x] Find out what to research and what to display
- [x] Research (partially done)
- [x] Script
- [x] Filming location (flat?)

**Comment by summerysaturn** [comment]
Found the media lab, we can book it out![IMG_20201016_150757.jpg](https://user-images.githubusercontent.com/48696244/96269513-5dff0c00-0fc2-11eb-952b-82dd4febdc77.jpg) ![IMG_20201016_150712.jpg](https://user-images.githubusercontent.com/48696244/96269522-60616600-0fc2-11eb-9c16-3a08b23effe2.jpg)

**Comment by summerysaturn** [comment]
New group member! We need to organise the planning for this asap, library time. Will look into lining up schedules if need be.

**Comment by summerysaturn** [comment]
shoot looks to be going ahead next week (WC 2020-11-09). got a good setup so far, lighting diagram looks good. Script and research are now urgent!!!

![image](https://user-images.githubusercontent.com/48696244/97874074-62405e80-1d10-11eb-82d5-2e0d5afb0a98.png)

quick note
- organising
- shooting
- assistant editor
- director
- gaffer
- assistant writer

**Comment by summerysaturn** [comment]
Booked for 9:30 Wednesday!

Need script done by Tuesday at the very latest.

**Comment by summerysaturn** [comment]
Filming done, waiting on Sham to do the edit. Will assist if possible. Left to do is the group work contribution report and that's it for this asmt!

**Comment by summerysaturn** [comment]
editing can take place over christmas so this is very safe for a pass. final point is to finish out the contribution document

**Comment by summerysaturn** [comment]
Closing this as the project is going to be finished soon. Just needs some gentle nudges in the direction of the team leader. Confident so far.

**Comment by summerysaturn** [comment]
Re-opening as this has become a priority issue due to missing a deadline. More details to come - need to discuss with tutors about ways forward.

**Comment by summerysaturn** [comment]
This is currently out of my hands. Edit seems to be on-track. Might need some help with the references. At the moment, this is low priority.

**Comment by summerysaturn** [comment]
Done!
